lib/Auth/utils.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * The functions, which are needed in more than one class, are listed below.
+ * @author Micha Preußer <micha.preusser@netknights.it>
+ */
+class sspmod_privacyidea_Auth_utils {
+
+	public function curl($params, $headers, $serverconfig, $url_ending, $method_post) {
```

Comment:
Just some nitpicking about the parameters:

1. could you please add some documentationf or the function / parameters

2. I recommend chosing another naming:
    * ``url_ending`` sounds a bit strange. I think something like *url_path* or maybe even better ``api_endpoint`` might be better.
    * The parameters would be more future robust, if we would call it ``http_method`` and expect values like
      ``POST`` or  ``GET``. Then - one day - the API would even allow to accept a ``DELETE`` method.

Reply:
See https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/46/commits/3c6c93f10f7204e1191a109d85724517f801adc4